x/ibc/light-clients/solomachine/types/header_test.go

```diff
@@ -21,27 +21,55 @@ func (suite *SoloMachineTestSuite) TestHeaderValidateBasic() {
 		{
 			"sequence is zero",
 			&types.Header{
-				Sequence:     0,
-				Signature:    header.Signature,
-				NewPublicKey: header.NewPublicKey,
+				Sequence:       0,
+				Timestamp:      header.Timestamp,
+				Signature:      header.Signature,
+				NewPublicKey:   header.NewPublicKey,
+				NewDiversifier: "",
+			},
+			false,
+		},
+		{
+			"timestamp is zero",
+			&types.Header{
+				Sequence:       header.Sequence,
+				Timestamp:      0,
+				Signature:      header.Signature,
+				NewPublicKey:   header.NewPublicKey,
+				NewDiversifier: header.NewDiversifier,
 			},
 			false,
 		},
 		{
 			"signature is empty",
 			&types.Header{
-				Sequence:     header.Sequence,
-				Signature:    []byte{},
-				NewPublicKey: header.NewPublicKey,
+				Sequence:       header.Sequence,
+				Timestamp:      header.Timestamp,
+				Signature:      []byte{},
+				NewPublicKey:   header.NewPublicKey,
+				NewDiversifier: header.NewDiversifier,
+			},
+			false,
+		},
+		{
+			"diversifier is blank",
```

Comment:
```suggestion
			"diversifier contains only spaces",
```
I think `blank` can be easily interpreted as `empty` so I think being explicit is best